src/index.html

```diff
@@ -7,8 +7,62 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="MOYO">
+      </a>
+      <nav class="nav_header">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav_item">
```

Comment:
```suggestion
      <nav class="nav_header">
        <ul class="nav_list">
          <li class="nav_item">
            <a href="#" class="nav_link is-active">
              Apple
            </a>
          </li>
          
          <li class="nav_item">
            <a href="#" class="nav_link">
              Samsung
            </a>
          </li>
          
          <li class="nav_item">
```
fix code style all over the page